File: accommodation-booking/server/src/Services/Identity/Identity.API/Integration/Events/TestIntegrationEvent.cs
using System.Text.Json.Serialization;

namespace Identity.API.Integration.Events
{
    public class TestIntegrationEvent : IntegrationEvent
    {
        public string Name { get; private set; }

        [JsonConstructor]
        public TestIntegrationEvent(string name) : base()
        {
            Name = name;
        }

    }
}

File: accommodation-booking/server/src/Services/Accomodation/src/main/java/accomodation/integration/events/TestIntegrationEvent.java
package accomodation.integration.events;

import accomodation.integration.IntegrationEvent;
import com.fasterxml.jackson.annotation.JsonProperty;

public class TestIntegrationEvent extends IntegrationEvent {
    private String name;

    public TestIntegrationEvent() {
        super();
    }

    public TestIntegrationEvent(String name) {
        super();
        this.name = name;
    }

    @JsonProperty("Name")
    public String getName() {
        return name;
    }

    @JsonProperty("Name")
    public void setName(String name) {
        this.name = name;
    }
}
